packages/api/src/moderation/util.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  AppBskyEmbedRecord,
+  AppBskyEmbedRecordWithMedia,
+  AppBskyFeedPost,
+} from '../client'
+import { ModerationDecision, ModerationUI } from './types'
+
+export function takeHighestPriorityDecision(
+  ...decisions: (ModerationDecision | undefined)[]
+): ModerationDecision {
+  // remove undefined decisions
+  const filtered = decisions.filter((d) => !!d) as ModerationDecision[]
+  if (filtered.length === 0) {
+    return ModerationDecision.noop()
+  }
+
+  // sort by highest priority
```

Comment:
Could be worth a note somewhere around here that "highest priority" means "least priority number".  I.e. that we want to sort lowest to highest.